spec/services/message_employee_matcher_spec.rb

```diff
@@ -18,6 +18,26 @@
       Timecop.return
     end
 
+    it "matches messages to users if the user's time has advanced past the scheduled message send at time but hasn't been sent yet" do
+      Timecop.freeze(Time.current)
+
+      days_after_start = 3
+      scheduled_message_time = Time.current.in_time_zone("Eastern Time (US & Canada)")
+      scheduled_message_time -= 1800
```

Comment:
This is one case where I tend to veer away from being verbose (as long as it is a simple statement like in this case), but I am fine changing it as several folks have mentioned the same preference about statements like this in previous teams I've been a part of.